oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/DocumentNodeStore.java

```diff
@@ -3429,7 +3429,7 @@ private String diffImpl(AbstractDocumentNodeState from, AbstractDocumentNodeStat
                 // see OAK-6016 and OAK-6011
                 if (e instanceof IllegalStateException &&
                         "Root document does not have a lastRev entry for local clusterId 0".equals(e.getMessage())) {
-                    LOG.warn("diffJournalChildren failed with " +
+                    LOG.debug("diffJournalChildren failed with " +
```

Comment:
It is more efficient to use the parametrized version of this method (with {} placeholders). As this log will rarely be printed, it is wasteful to create the String that is passed as an argument.

Reply:
+1